From f8c17da03af232f3287b26cac054ac3b83f5361f Mon Sep 17 00:00:00 2001
From: Xose Vazquez Perez <xose.vazquez@gmail.com>
Date: Mon, 24 Jun 2013 12:41:13 +0200
Subject: [PATCH 63/65] r92su: fixed wrong usb ids

{USB_DEVICE(0x0BDA, 0x8715)} should be
{USB_DEVICE(0x0BDA, 0x8175)}
                       ^^^

{USB_DEVICE(0x0009, 0x21E8)} should be
{USB_DEVICE(0x0009, 0x21E7)}
                         ^
Cc: Christian Lamparter <chunkeey@googlemail.com>
Signed-off-by: Xose Vazquez Perez <xose.vazquez@gmail.com>
---
 r92su/usb.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/wireless/r92su/usb.c
+++ b/drivers/net/wireless/r92su/usb.c
@@ -477,7 +477,7 @@ static struct usb_device_id r92su_usb_pr
 
 /* RTL8191SU */
 	/* Realtek */
-	{USB_DEVICE(0x0BDA, 0x8715)},
+	{USB_DEVICE(0x0BDA, 0x8175)},
 	{USB_DEVICE(0x0BDA, 0x8172)},
 	{USB_DEVICE(0x0BDA, 0x8192)},
 	/* Airlive */
@@ -551,7 +551,7 @@ static struct usb_device_id r92su_usb_pr
 
 
 /* Unknown Chip */
-	{USB_DEVICE(0x0009, 0x21E8)},
+	{USB_DEVICE(0x0009, 0x21E7)},
 	{USB_DEVICE(0x0E0B, 0x9063)},
 	{USB_DEVICE(0x5A57, 0x0291)},
 	{ },
